### PR TITLE
Add codescene.io analysis results

### PR DESCRIPTION
**Pull request by wachulski**
Codescene.io doesn't require any build programming, 
just a free account and an integration button click at codescene.io.
This may be a perfect fit for codecov analysis already in place.

### CODE REVIEW

**Comment by spdr870** [comment]
@wachulski I looked over the results in the sample. Nice results. However, if I look at the plans offered by codescene.io it lookes like we do not fit the free plan. The payed plans are expensive. How do you see this?

**Comment by gerhardol** [comment]
codescene has a nice interface to find refacoring targets. I do not find that so important myself, other services have similar information.

SonarCube and Codacy has the statical analysis and helps finds issues. Of use when analysing PRs.

**Comment by wachulski** [comment]
@spdr870 could you elaborate on where you actually see the limitation that hurts us? For me https://codescene.io/plans `Free` has just a shallow history (5000 last commits only).

**Comment by spdr870** [comment]
> @spdr870 could you elaborate on where you actually see the limitation that hurts us? For me https://codescene.io/plans `Free` has just a shallow history (5000 last commits only).

When only importing the last 5000 commits, there is no problem indeed.

**Comment by RussKie** [comment]
@gitextensions/git-extensions-source taking in?

File: README.md [hunk 1]
```diff
@@ -172,3 +172,7 @@ Support this project by becoming a sponsor. Your logo will show up here with a l
 Watch us work on this project.
 
 [![Click to play](https://img.youtube.com/vi/k6DhHbzkZvM/0.jpg)](https://www.youtube.com/watch?v=k6DhHbzkZvM)
+
+# Code Quality Monitor
+
+[![Code quality status](https://codescene.io/projects/4180/status.svg) Full analysis results at **codescene.io**.](https://codescene.io/projects/4180/jobs/latest-successful/results)
```
Comment (RussKie) [inline README.md]:
```suggestion
[![Code quality status](https://codescene.io/projects/5326/status.svg) Full analysis results at **codescene.io**.](https://codescene.io/projects/5326/jobs/latest-successful/results)
```

**Comment by RussKie** [comment]
Thank you

**Pull request merged**